### PR TITLE
chore: Reduce CA default lifetime to 1 year

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

Part of #358 

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: rust/operator-binary/src/backend/tls/mod.rs [hunk 1]
```diff
@@ -43,7 +43,7 @@ mod ca;
 
 /// How long CA certificates should last for. Also used for calculating when they should be rotated.
 /// Must be less than half of [`DEFAULT_MAX_CERT_LIFETIME`].
```
Comment (NickLarsenNZ) [inline rust/operator-binary/src/backend/tls/mod.rs:45]: I believe this comment is wrong, but I don't know what is should be replaced with.

**Review by maltesander — COMMENTED** [review]
That could deserve a changelog entry :) ?

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by lfrancke** [comment]
Is this documented somewhere?

**Comment by fhennig** [comment]
maybe this one? https://github.com/stackabletech/secret-operator/pull/441

**Pull request merged**